src/Task/FileSystem/CopyDir.php

```diff
@@ -32,7 +32,9 @@ protected function copyDir($src, $dst)
         if (false === $dir) {
             throw new TaskException($this, "Cannot open source directory '" . $src . "'");
         }
-        @mkdir($dst);
+        if (!is_dir($dst)) {
+            mkdir($dst, 0777, true);
```

Comment:
I did not want to change existing behavior, and 0777 is the default (http://php.net/manual/de/function.mkdir.php).